artemis/frontend.py

```diff
@@ -174,8 +180,28 @@ async def post_add(
             },
             csrf_protect,
         )
+    if schedule_enabled and schedule_interval and schedule_start:
+        try:     
+            start_time = datetime.fromisoformat(schedule_start)
+            end_time = datetime.fromisoformat(schedule_end)
+        except:
+            raise HTTPException(status_code=400, detail="Invalid start time format")
+
+        schedule_periodic_scan(
+            targets=total_list,
+            tag=tag,
+            disabled_modules=disabled_modules,
+            priority=TaskPriority(priority),
+            interval_minutes=schedule_interval,
+            start_time=start_time,
+            end_time=end_time
+        )
+
+    else:
```

Comment:
what do you think about adding the possiblity to change and cancel the scheduled san?